### ISSUE TITLE
Cannot read property 'extensions' of undefined — after upgrading from 1.9.1 to 1.13.2

### ISSUE THREAD

**Issue by alamothe** [dotansimha/graphql-code-generator#3865]
**Describe the bug**

Running:
```
./node_modules/.bin/graphql-code-generator
```
Fails with:
```
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate src/fragmentTypes.json
      ✖ Load GraphQL schemas
        → Failed to load schema
        Load GraphQL documents
        Generate
    ❯ Generate src/graphql-types.ts
      ✖ Load GraphQL schemas
        → Failed to load schema
        Load GraphQL documents
        Generate


 Found 2 errors

  ✖ src/fragmentTypes.json
    Failed to load schema from src/schema.json:

        Cannot read property 'extensions' of undefined
        TypeError: Cannot read property 'extensions' of undefined
    at applyExtensionObject (/Users/nidza/Projects/repro/node_modules/@graphql-toolkit/schema-mergin
g/index.cjs.js:834:41)
    at applyExtensions (/Users/nidza/Projects/repro/node_modules/@graphql-toolkit/schema-merging/ind
ex.cjs.js:840:9)
    at makeSchema (/Users/nidza/Projects/repro/node_modules/@graphql-toolkit/schema-merging/index.cj
s.js:951:5)
    at Object.mergeSchemasAsync (/Users/nidza/Projects/repro/node_modules/@graphql-toolkit/schema-me
rging/index.cjs.js:921:12)
```

**To Reproduce**

I made a repo with repro: https://github.com/alamothe/gcg-repro

**Expected behavior**

Used to work in 1.9.1.

**Environment:**

- OS: MacOS
- `@graphql-codegen/...`: 
```
    "@graphql-codegen/cli": "1.13.2",
    "@graphql-codegen/fragment-matcher": "1.13.2",
    "@graphql-codegen/typescript": "1.13.2",
    "@graphql-codegen/typescript-operations": "1.13.2"
```
- NodeJS: v10.19.0

**Additional context**

**Comment by ardatan** [comment]
What version of GraphQL-js are you using? Because it is missing in the repo.

**Comment by alamothe** [comment]
Not sure what you're asking. I can repro just with those dependencies, without specifying any other dependency such as `graphql`. So I suppose I use whatever is imported by your library. Running "npm list" gives me:
```
~/Projects/repro $ npm list graphql
repro@1.0.0 /Users/nidza/Projects/repro
└─┬ @graphql-codegen/cli@1.13.2
  └─┬ @graphql-toolkit/apollo-engine-loader@0.10.3
    └─┬ apollo-language-server@1.21.0
      └── graphql@14.6.0 
```
So looks like you import graphql@14.6.0.

What do you mean by "log doesn't match the reproduction."?

**Comment by ardatan** [comment]
@alamothe Sorry that was my mistake. Normally, you need to install `graphql` by yourself. Could you install `graphql` and try again?

**Comment by alamothe** [comment]
Which version do you want me to install? Like I said, your package is importing `graphql@14.6.0` as a dependency so that version is already installed.

**Comment by ardatan** [comment]
Could you install latest version of graphql and try again because that dependency is indirect implicit dependency?
We will remove that on next release.

**Comment by melounek** [comment]
Heh, I tried to reproduce other issue ( #3560 ) But ended up reproducing this one, so I tagged that repo-state hoping it could help: https://github.com/melounek/repro-codegen/tree/issue-3865

**Comment by dotansimha** [comment]
Thanks @melounek , I managed to find the cause for this. Fixing it now!

**Comment by dotansimha** [comment]
Fixed in `v1.14.0`.

**Comment by alamothe** [comment]
Thank you so much for fixing this!
